src/HyperFormula.ts

```diff
@@ -293,8 +293,8 @@ export class HyperFormula implements TypedEmitter {
    *
    * @category Factories
    */
-  public static buildEmpty(configInput?: Partial<ConfigParams>): HyperFormula {
-    return this.buildFromEngineState(BuildEngineFactory.buildEmpty(configInput))
+  public static buildEmpty(configInput: Partial<ConfigParams> = {}, namedExpressions: SerializedNamedExpression[] = []): HyperFormula {
```

Comment:
Is it a good choice for `namedExpressions`? I think that by similarity to `functionPlugins` the should be placed within Config object.

On the other hand, Config is not a user content. It may be stored separately and be a constant. Sometimes hardcoded into the app. When we look at it like that, it makes sense to separate those contexts.

@izulin You probably thought that through already, WDYT?

Reply:
I think it should go other way around: If there was a way to hardcode it into the actual data (like in `buildFromArray` or `buildFromSheets`), it should be. But I cannot think of a way to make it so consistently without making unnecessary layers of complexity, so I think keeping it as a separate argument should do fine.

Reply:
Yup. And having it in Config will complicate updateConfig logic. Good choice